tools/rpm2img

```diff
@@ -219,18 +219,22 @@ dd if="${BOTTLEROCKET_DATA}" of="${DATA_IMAGE}" conv=notrunc bs=1M seek=1
 sgdisk -v "${DISK_IMAGE}"
 sgdisk -v "${DATA_IMAGE}"
 
+IMAGE_EXTENSION="img.lz4"
 if [[ ${OUTPUT_FMT} == "raw" ]]; then
   lz4 -vc "${DISK_IMAGE}" >"${OUTPUT_DIR}/${DISK_IMAGE_BASENAME}.img.lz4"
   lz4 -vc "${DATA_IMAGE}" >"${OUTPUT_DIR}/${DATA_IMAGE_BASENAME}.img.lz4"
-  chown 1000:1000 "${OUTPUT_DIR}/${DISK_IMAGE_BASENAME}.img.lz4" \
-    "${OUTPUT_DIR}/${DATA_IMAGE_BASENAME}.img.lz4"
+elif [[ ${OUTPUT_FMT} == "qcow2" ]]; then
+  qemu-img convert -f raw -O qcow2 "${DISK_IMAGE}" "${OUTPUT_DIR}/${DISK_IMAGE_BASENAME}.qcow2"
+  qemu-img convert -f raw -O qcow2 "${DATA_IMAGE}" "${OUTPUT_DIR}/${DATA_IMAGE_BASENAME}.qcow2"
+  IMAGE_EXTENSION="qcow2"
 elif [[ ${OUTPUT_FMT} == "vmdk" ]]; then
   # Stream optimization is required for creating an Open Virtual Appliance (OVA)
   qemu-img convert -f raw -O vmdk -o subformat=streamOptimized "${DISK_IMAGE}" "${OUTPUT_DIR}/${DISK_IMAGE_BASENAME}.vmdk"
   qemu-img convert -f raw -O vmdk -o subformat=streamOptimized "${DATA_IMAGE}" "${OUTPUT_DIR}/${DATA_IMAGE_BASENAME}.vmdk"
-  chown 1000:1000 "${OUTPUT_DIR}/${DISK_IMAGE_BASENAME}.vmdk" \
-    "${OUTPUT_DIR}/${DATA_IMAGE_BASENAME}.vmdk"
+  IMAGE_EXTENSION="vmdk"
 fi
+chown 1000:1000 "${OUTPUT_DIR}/${DISK_IMAGE_BASENAME}.${IMAGE_EXTENSION}" \
+  "${OUTPUT_DIR}/${DATA_IMAGE_BASENAME}.${IMAGE_EXTENSION}"
```

Comment:
We can just `chown` everything under `${OUTPUT_DIR}` - then we don't need to keep track of `IMAGE_EXTENSION`, and we only need to run it once at the very end.

```suggestion
find "${OUTPUT_DIR}" -type f -print -exec chown 1000:1000 {} \;
```

Reply:
Done.